lib/util/browserdetection.js

```diff
@@ -27,6 +27,15 @@ function isChromeIOS() {
   return guessBrowser() === 'chrome' && isMobile() && isIOS();
 }
 
+/**
+ * Check whether the current OS version is 14_3 and above,
+ * used in conjunction with isChromeIOS and isMobile.
+ * @returns {boolean}
+ */
+function isChromeVersionSupported() {
+  return /14_([3-9]|[1-9]\d+)|(1[5-9]|[2-9]\d+)_\d+/.test(navigator.userAgent);
+}
```

Comment:
@PikaJoyce Do we need this check? Since WebRTC APIs are not available in Chrome on iOS 14.2 and below, doesn't `isWebRTCSupported()` already return false?